File: rest/participant/instance-get-example-1/instance-get-example-1.5.x.cs
// Download the twilio-csharp library from twilio.com/docs/libraries/csharp
using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account.Conference;

class Example
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Auth Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";
        TwilioClient.Init(accountSid, authToken);

        const string conferenceSid = "CFbbe4632a3c49700934481addd5ce1659";
        const string callSid = "CA386025c9bf5d6052a1d1ea42b4d16662";
        var participant = ParticipantResource.Fetch(conferenceSid, callSid);

        Console.WriteLine(participant.StartConferenceOnEnter);
    }
}

File: rest/applications/instance-get-example-1/instance-get-example-1.5.x.py
# Download the Python helper library from twilio.com/docs/python/install
from twilio.rest import TwilioRestClient

# Your Account Sid and Auth Token from twilio.com/user/account
account_sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
auth_token = "your_auth_token"
client = TwilioRestClient(account_sid, auth_token)

app = client.applications.get("AP2a0747eba6abf96b7e3c3ff0b4530f6e")
print(app.sms_url)
